src/app/registration/registration.component.css

```diff
@@ -2,17 +2,21 @@ a {
   color: #459fed;
   text-decoration: none;
 }
+
 .title1 {
   display: block;
```

Comment:
First of, use flex to position the element. Second, do not use !important. There are better ways to center the content.